go/vt/vtgate/planbuilder/postprocess.go

```diff
@@ -140,6 +140,9 @@ func pushLimit(limit *sqlparser.Limit, bldr builder) (builder, error) {
 		return bldr, nil
 	}
 	lb := newLimit(bldr)
+	if limit != nil && limit.Offset != nil && len(orderBy) == 0 {
+		return nil, errors.New("unsupported: offset limit for cross-shard queries")
```

Comment:
This error should be `offset limit for cross-shard queries without order by` right?
I think if we want to enforce this, we should apply the rule even if offset is not specified.

I'm thinking we should not enforce this. If people don't want ordering, it's their problem.